superset/db_engine_specs/bigquery.py

```diff
@@ -39,7 +43,19 @@
 )
 
 
-class BigQueryEngineSpec(BaseEngineSpec):
+class BigQueryParametersSchema(Schema):
+    credentials_json = fields.Dict(
```

Comment:
Should we be more specific here, to leverage Marshmallow validation? The credentials looks like this:

```json
{
  "type": "service_account",
  "project_id": "optimal-aurora-285000",
  "private_key_id": "a1a0f6e452b06c145878f283fcb2d252759b97a3",
  "private_key": "-----BEGIN PRIVATE KEY-----\nXXX\n-----END PRIVATE KEY-----\n",
  "client_email": "superset@optimal-aurora-285000.iam.gserviceaccount.com",
  "client_id": "114899578697537931695",
  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  "token_uri": "https://oauth2.googleapis.com/token",
  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/superset%40optimal-aurora-285000.iam.gserviceaccount.com"
}
```